R/sha1.R

```diff
@@ -449,3 +449,5 @@ sha1.formula <- function(x, digits = 14L, zapsmall = 7L, ..., algo = "sha1"){
     )
     digest(y, algo = algo)
 }
+
+"sha1.(" <- sha1.formula
```

Comment:
So ... that line is the gist of the PR?  Kewl.

Reply:
Yep, it's pretty simple.

Reply:
Elegant solution